client/signup/steps/user/index.jsx

```diff
@@ -118,6 +119,20 @@ export class UserStep extends Component {
 		}
 	}
 
+	componentDidUpdate() {
+		if ( this.userCreationCompletedWithAndHasHistory( this.props ) ) {
+			// It looks like the user just completed the User Registartion Step
+			// And clicked the back button. Lets redirect them to the this page but this time they will be logged in.
+			const url = new URL( window.location );
+			const search_params = url.searchParams;
```

Comment:
Why do we need a new variable for `search_params`? Also, let's keep in mind the coding standards and the variable naming conventions (it should be `searchParams` if we decide to keep it).

Reply:
I updated the variable to use `searchParams`. 
Not sure if there is a better way to write the same thing though.